src/BizHawk.Emulation.Cores/Consoles/Nintendo/N64/N64SyncSettings.GLideN64.cs

```diff
@@ -11,6 +11,7 @@ public N64GLideN64PluginSettings()
 			{
 				UseDefaultHacks = true;
 
+				BackgroundsMode = 1;
```

Comment:
BackgroundsMode is type BackgroundsRenderingMode, not type int. This is causing the build error. Replace with this:
```suggestion
				BackgroundsMode = BackgroundsRenderingMode.Stripped;
```